Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/tracing/card/TracingCardState.kt

```diff
@@ -272,13 +269,15 @@ data class TracingCardState(
     fun formatRiskLevelHeadline(c: Context, riskLevelScore: Int): String {
         return if (tracingStatus != GeneralTracingStatus.Status.TRACING_INACTIVE) {
             when (riskLevelScore) {
-                RiskLevelConstants.INCREASED_RISK -> R.string.risk_card_increased_risk_headline
-                RiskLevelConstants.UNKNOWN_RISK_OUTDATED_RESULTS -> R.string.risk_card_outdated_risk_headline
+                RiskLevelConstants.INCREASED_RISK ->
+                    R.string.risk_card_increased_risk_headline
+                RiskLevelConstants.UNKNOWN_RISK_OUTDATED_RESULTS,
+                RiskLevelConstants.UNKNOWN_RISK_OUTDATED_RESULTS_MANUAL ->
```

Comment:
What do we need this for?

Reply:
While these two are also used in relation to the grey risk card, they are also needed for displaying the "Risiko-Ermittlung
nicht möglich" card which is a white card from what I can see in the Android Figma file. AFAIK there are instances where the risk calculation can fail after it's been calculated at least once, so these constants should be relevant for the cards with the white background (at least for now).